tests/Session/test_commands.py

```diff
@@ -100,6 +104,61 @@ def testHelp(self):
 
         # reset standout
         sys.stdout = sys.__stdout__
+    
+    @patch('src.Session.commands.input', create=True)
+    def testOpenFoodStorage(self, mocked_input):
+        # grab print output
+        capturedPrintOutput = io.StringIO()
+        sys.stdout = capturedPrintOutput
+
+        self.commands.create_foodStorage(freezer_name)
+        self.commands.open(freezer_name)
+
+        expectedOutput = OPEN_FOOD_STORAGE_MESSAGE.format(freezer_name)
+        self.assertEqual(expectedOutput, capturedPrintOutput.getvalue())
+        
+        # reset standout
+        sys.stdout = sys.__stdout__
+
+        ## Happy path: opens food storage with existing name ##
+        mocked_input.side_effect = [freezer_name, foodstorage_name]
+        self.commands.open(freezer_name)
+
+        expectedOutput = OPEN_FOOD_STORAGE_SUCCESS_MESSAGE.format(freezer_name, foodstorage_name)
+        self.assertEqual(expectedOutput, capturedPrintOutput.getvalue())
```

Comment:
So I think these two can be combined, and they're almost there, but slightly off.

You should use side_effect to create a foodStorage and then open it. And then you can also check the prints like you've been doing. You just gotta move side_effect up and make sure it includes all the answers you need for `create_foodStorage` and `open`

Reply:
I followed your suggestions and I also added the test you did in the beginning of `testCreateFoodStorage` to test if there is no profile available since `open` checks for that as well. Let me know if that was unnecessary and if I did not implement your suggestions correctly! Thanks!